core/src/main/java/com/linecorp/armeria/server/Http1RequestDecoder.java

```diff
@@ -261,7 +261,8 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
                                             eventLoop, id, 1, headers, false, inboundTrafficController,
                                             serviceConfig.maxRequestLength(), routingCtx,
                                             ExchangeType.BIDI_STREAMING,
-                                            System.nanoTime(), SystemInfo.currentTimeMicros(), true, false);
+                                            System.nanoTime(), SystemInfo.currentTimeMicros(), true, false,
+                                            cfg.serverMetrics());
```

Comment:
... which means we could just increase the counter in this class, whichever `DecodedHttpRequest` type we instantiate?